internal/pkg/service/buffer/config/worker.go

```diff
@@ -42,18 +44,22 @@ func NewWorkerConfig() WorkerConfig {
 		ServiceConfig:           NewServiceConfig(),
 		UniqueID:                "",
 		CheckConditionsInterval: DefaultCheckConditionsInterval,
-		UploadConditions:        DefaultUploadConditions(),
-		MetricsListenAddress:    "0.0.0.0:9000",
-		ConditionsCheck:         true,
-		CloseSlices:             true,
-		UploadSlices:            true,
-		RetryFailedSlices:       true,
-		CloseFiles:              true,
-		ImportFiles:             true,
-		RetryFailedFiles:        true,
-		TasksCleanup:            true,
-		TasksCleanupInterval:    DefaultCleanupInterval,
-		UploadTransport:         nil, // use default transport
+		UploadConditions: Conditions{
+			Count: 1000,
+			Size:  1 * datasize.MB,
+			Time:  1 * time.Minute,
+		},
```

Comment:
Prvy commit, zmazal som `DefaultUploadConditions` funkciu, kedze sa pouziva iba raz a bola v inom subore ako ostatne konstanty.

Reply:
Staci,ze je splnena jedna z tychto podmienok a spusti sa upload.

V nasom benchmarku to trigruje `Size>1MB` a `Count>1000`.

Problem bol v tom, ze tato podmienka sa kontroluje kazdych `30s`, takze castejsie upload nastat nemohol.